blog/content/post/2023-06-23-Blogpost_yocto_kas.md

```diff
@@ -0,0 +1,286 @@
+---
+title: Blog post - odświeżenie "Minimal Image, Fastest Rpi, Quick Tutorial"
+abstract: 'So far, to build even a minimal image on th raspberry pi the downloading of sources and then configuration had to be done by hand.
+	  Instead Yocto kas is using a project configuration file and does the download and configuration phase.
+	  Kas tool provides an easy mechanism to setup bitbake based projects.
+          Kas makes the setup of a Yocto build environment super simple and super fast.'
```

Comment:
@Kewkaa please wrap at 80 characters, layout of the blog post kind of suggest you don't have your editor configured correctly.

@macpijan shouldn't pre-commit detect this problem?